aioseop_class.php

```diff
@@ -919,6 +919,9 @@ public function get_title_format( $args ) {
 		$w            = $info['w'];
 		$p            = $info['p'];
 
+		if ( strpos( $title_format, '%site_title%' ) !== false ) {
+			$title_format = str_replace( '%site_title%', get_bloginfo( 'name' ), $title_format );
+		}
 		if ( strpos( $title_format, '%blog_title%' ) !== false ) {
```

Comment:
It might help to add a dev note here that %blog_title% is deprecated.

Reply:
Done.